Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/NodeFunctionGeneric.h

```diff
@@ -88,14 +88,17 @@ namespace ScriptCanvas
             {\
                 return GetName(i);\
             }\
-            AZStd::string_view argName = GetName(i);\
-            if (!argName.empty())\
-            {\
-                return argName;\
-            }\
             else\
             {\
-                return AZStd::string::format("Input [%zu]", i);\
```

Comment:
Minor: is this change behaving same as before? as when argument number is 0 or 1, it will return directly

Reply:
Yes, the diff is strange thought, doesn't show the important part of the changes.